packages/infra/src/index.ts

```diff
@@ -0,0 +1,533 @@
+import path from 'path'
```

Comment:
For my own information. Was this file auto-generated or hand crafted? That's a large file :)

Reply:
It was not auto-generated.

Reply:
Wow. That's a large config - not impressed by Beanstalk ease of configuration...

Reply:
It's mostly because CDK doesn't really support beanstalk yet.  It's easier using the Web console, I guess.  I dislike beanstalk for the same reasons I dislike Heroku.  Mostly just trying something out.  Chance I might dump it at some point...

Reply:
We discussed during the offsite that we'd like to user serverless + CDK when we migrate the launchpad stack. A few folks in the team have been experimenting with https://sst.dev and have good things to say about it. Seems simpler than beanstalk.